stable/admin/templates/statefulset.yaml

```diff
@@ -111,11 +111,13 @@ spec:
         volumeMounts:
         - name: log-volume
           mountPath: /var/log/nuodb
-{{- if hasKey .Values.admin.configFiles "nuodb.lic" }}
-        - name: nuodb-license-volume
-          mountPath: /etc/nuodb/nuodb.lic
-          subPath: nuodb.lic
-{{ end }}
+        {{- if .Values.admin.configFiles }}
+        {{- range $key, $val := .Values.admin.configFiles }}
```

Comment:
Minor point:
For this kind of idiom, I prefer:

{{- with .Values.admin.configFiles }}
{{ - range $key, $val := . }}
It avoids repeating the .Values reference, and avoids bugs - either now or later - where the 2 references diverge.